src/modules/vendor/components/questionnaires/ProfileEdition.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <ni-input v-model.trim="questionnaire.title" required-field caption="Titre" @blur="updateQuestionnaireTitle"
+    @focus="saveTmpTitle" :error="$v.questionnaire.title.$error" :disable="questionnaire.status === PUBLISHED" />
+</template>
+
+<script>
+import get from 'lodash/get';
+import { mapState } from 'vuex';
+import { required } from 'vuelidate/lib/validators';
+import Questionnaires from '@api/Questionnaires';
+import { NotifyNegative, NotifyWarning, NotifyPositive } from '@components/popup/notify';
+import Input from '@components/form/Input';
+import { PUBLISHED } from '@data/constants';
+
+export default {
+  name: 'ProfileEdition',
+  props: {
+    profileId: { type: String, required: true },
+  },
+  components: {
+    'ni-input': Input,
+  },
+  validations () {
+    return {
+      questionnaire: { title: { required } },
+    };
+  },
+  data () {
+    return {
+      tmpInput: '',
+      PUBLISHED,
+    };
+  },
+  computed: {
+    ...mapState('questionnaire', ['questionnaire']),
+
+  },
+  async created () {
+    await this.refreshQuestionnaire();
+  },
+  methods: {
+    async refreshQuestionnaire () {
+      try {
+        await this.$store.dispatch('questionnaire/fetchQuestionnaire', { questionnaireId: this.profileId });
+      } catch (e) {
+        console.error(e);
+      }
+    },
+    saveTmpTitle () {
+      this.tmpInput = get(this.questionnaire, 'title') || '';
```

Comment:
`get(this.questionnaire, 'title', '')` ne fonctionne pas ?
Tu pourrais le passer directement en props (sans methods), non ?